src/ert/_c_wrappers/enkf/site_config.py

```diff
@@ -63,106 +116,47 @@ def from_config_dict(self, config_dict):
                     config_file=job[ConfigKeys.PATH],
                     private=False,
                     name=job[ConfigKeys.NAME],
-                    license_root_path=__license_root_path,
+                    license_root_path=license_root_path,
                 )
                 new_job.convertToCReference(None)
                 ext_job_list.add_job(job[ConfigKeys.NAME], new_job)
-            except (ValueError, OSError):
-                print(f"WARNING: Unable to create job from {job[ConfigKeys.PATH]}")
+            except (ValueError, OSError) as e:
+                print(f"WARNING: Unable to create job from {job[ConfigKeys.PATH]}: {e}")
 
         for job_path in config_dict.get(ConfigKeys.INSTALL_JOB_DIRECTORY, []):
             if not os.path.isdir(job_path):
                 print(f"WARNING: Unable to locate job directory {job_path}")
                 continue
             files = os.listdir(job_path)
+            print(files)
             for file_name in files:
                 full_path = os.path.join(job_path, file_name)
                 if os.path.isfile(full_path):
                     try:
                         new_job = ExtJob(
                             config_file=full_path,
                             private=False,
-                            license_root_path=__license_root_path,
+                            license_root_path=license_root_path,
                         )
                         new_job.convertToCReference(None)
                         ext_job_list.add_job(new_job.name(), new_job)
                     except (ValueError, OSError):
-                        print(f"WARNING: Unable to create job from {full_path}")
+                        print(
```

Comment:
just rethrow

Reply:
also in new line 131. may i also suggest to make this into a function? by this i mean this block of taking a file name, and creating / adding an ExtJob. so from 142 to 154, and then hopefully also from 121 to 131, might be refactored into one function.

Reply:
@valentin-krasontovitsch The two paths are currently too different to be made neatly into one function. Will probably be possible after further refactoring, in the next PR.

Reply:
fair enough!